fhttp/http_server.go

```diff
@@ -97,17 +102,25 @@ func EchoHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
-// MaxPayloadSize is the maximum size of payload to be generated by the
-// EchoHandler size= argument. In bytes.
-const MaxPayloadSize = 256 * 1024 // 256k
-
-var payload = make([]byte, MaxPayloadSize)
-
 func init() {
+	ChangeMaxPayloadSize(MaxPayloadSize)
+}
+
+// ChangeMaxPayloadSize is used to change max payload size and fill it with pseudorandom content
+func ChangeMaxPayloadSize(newMaxPayloadSize int) {
+	if newMaxPayloadSize >= 0 {
+		MaxPayloadSize = newMaxPayloadSize
+	} else {
+		MaxPayloadSize = 0
+	}
+	payload = make([]byte, MaxPayloadSize)
 	// One shared and 'constant' (over time) but pseudo random content for payload
 	// (to defeat compression). We don't need crypto strength here, just low cpu
 	// and speed:
-	rand.Read(payload) // nolint:gas
+	_, err := rand.Read(payload) // nolint:gas
```

Comment:
(nit/optional) you can probably remove the nolint comment now that you check for err (also I gave up on gas anyway)

Reply:
I'm not sure how gas works. I copied it from existing

Reply:
I think I used to get 'unsafe use of rand' error if I didn't use // nolint:gas this execption

Reply:
let me try again removing the - // nolint:gas